pkg/services/preference/prefapi/api.go

```diff
@@ -58,7 +63,7 @@ func UpdatePreferencesFor(ctx context.Context,
 
 func GetPreferencesFor(ctx context.Context,
 	dashboardService dashboards.DashboardService, preferenceService pref.Service,
-	orgID, userID, teamID int64) response.Response {
+	orgID, userID, teamID int64, features featuremgmt.FeatureToggles) response.Response {
```

Comment:
I think we should reorder the arguments - the 'services' could come first.

```go
func GetPreferencesFor(ctx context.Context,
	dashboardService dashboards.DashboardService, preferenceService pref.Service, features featuremgmt.FeatureToggles,
	orgID, userID, teamID int64) response.Response {
```

Call sites will need to be updated.